draft-packages/collapsible/KaizenDraft/ExpertAdviceCollapsible/ExpertAdviceCollapsible.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from "@testing-library/react"
+import { configure, queryByTestId } from "@testing-library/dom"
+import * as React from "react"
+import { ExpertAdviceCollapsible } from ".."
+
+configure({ testIdAttribute: "data-automation-id" })
+
+it("renders the title as a heading", () => {
+  const { getByTestId } = render(
+    <ExpertAdviceCollapsible id="1" title="This title should be rendered">
+      Expert advice collapsible component
+    </ExpertAdviceCollapsible>
+  )
+  const titleText = getByTestId("collapsible-header-1").querySelector("div")
+  expect(titleText).toHaveTextContent("This title should be rendered")
+})
+
```

Comment:
I think adding one more test to verify that it take children and renders that to the page would be a good way to ensure it is correctly consuming the Collapsible. We could potentially use getByText again to validate that :)

Reply:
I had a go at this and am keen for feedback.